etc/docker/dev/README.rst

```diff
@@ -8,6 +8,15 @@ Prerequisites
 Setting up a Rucio development environment requires to have Docker installed. Docker is an
 application that makes it simple and easy to run application processes. To install Docker for
 your platform, please refer to the `Docker installation guide <https://docs.docker.com/install/>`_.
+After installation of docker, you can set up a virtual environment for developing Rucio : ::
+
+    virtualenv dev
+    source dev/bin/activate
+    (dev) pip install -r requirements.readthedocs.txt
+
```

Comment:
The virtual env here is not running in docker, right ? To have a local env we have the script `./tools/install_venv.py` which creates a local .venv directory and install all the dependencies (./tools/pip-requires*) but this won't get the server (running in the container).

Reply:
No, the virtualenv was on the host, so this comment is valid.

Reply:
I see, thanks for the clarification. Could you try the script `./tools/install_venv.py`  then ? I think it should achieve what you want (?)